authorizations/authorization-chain/src/test/java/org/apache/gravitino/authorization/chain/integration/test/TestChainedAuthorizationIT.java

```diff
@@ -307,11 +367,6 @@ void testChangeOwner() throws InterruptedException {
     // TODO
   }
 
-  @Test
```

Comment:
Why do we remove this test case?

Reply:
I only temporarily disable this test case, I will recover it in next PR.